palm/plugins/dbt/dbt_version_detection.py

```diff
@@ -0,0 +1,63 @@
+import click
+from pathlib import Path
+import yaml
+
+from palm.utils import run_in_docker
+
+def get_dbt_version() -> str:
+    """Get the dbt version from the dbt-core package in the docker image.
+
+    Returns:
+        str: The dbt version
+    """
+    image_name = _get_image_name()
+    cmd = "pip list | grep dbt-core"
```

Comment:
Nice! I thought about something like this too but forgot about pip list! Pretty clean.